src/views/help/PageHelp.vue

```diff
@@ -85,6 +95,14 @@ import { useQuery } from "@/util/composables";
 const { query, data: uptimes, isLoading, isError } = useQuery(getUptimes, []);
 
 onMounted(query);
+
+/** clear user localstorage data */
+function clearData() {
```

Comment:
I'd prefer for there to be a prompt before the local data is cleared, e.g. if the user accidentally hit the button. Even just calling the built-in `prompt()` function, since you're also using the built-in`alert()`, would IMO be fine.